float64 x
float64 y
float64 z
float64 yaw
float64 vx
float64 vy
float64 vz
float64 yaw_rate
float64 t
